airflow/providers/cloudant/provider.yaml

```diff
@@ -25,7 +25,7 @@ state: ready
 source-date-epoch: 1723969866
 # note that those versions are maintained by release manager - do not update them manually
 versions:
-  - 3.6.0
```

Comment:
Some provider tests were failing with error messages about 3.6.0 not being a valid revision, trace below. We have a tag for providers-cloudant/3.6.0rc1 but not for 3.6.0 GA; is it legit to change the newest version to 4.0.0, will that make the release process work?

```
New version of the 'cloudant' package is ready to be released!
Running command: <git log apache-https-for-providers/main...providers-cloudant/3.6.0 .>
Working directory <project-dir>/airflow/airflow/providers/cloudant

git log '--pretty=format:%H %h %cd %s' --date=short apache-https-for-providers/main...providers-cloudant/3.6.0 -- .

========================= STDERR start ============================
fatal: bad revision 'apache-https-for-providers/main...providers-cloudant/3.6.0'

```

Reply:
It will have to wait for release of 3.6.0 to happen

Reply:
(2 days or so)